pontoon/base/templatetags/helpers.py

```diff
@@ -252,10 +252,7 @@ def format_timedelta(value):
 @library.filter
 def nospam(self):
     return jinja2.Markup(
-        cgi.escape(self, True)
-        .replace("@", "&#64;")
-        .replace(".", "&#46;")
-        .replace("'", "&quot;")
```

Comment:
I removed this because the `escape` function from the `html` module encodes this character as an HTML entity. 
Also, there was a small bug in the previous function because `'` should be encoded as `&apos;` (I've learned this 3 days ago: D).

Reply:
Ah, you're using the `html` library now.

Thanks for the explanation and the bugfix! :)